pyproject.toml

```diff
@@ -62,7 +62,6 @@ plugins = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--cov las_geoh5 --cov-report html --cov-report term-missing:skip-covered"
```

Comment:
Can you bring this back, and remove the `fail_under = 80` below? 
I can show you how to make it pass with pycharm.